package.json

```diff
@@ -7,10 +7,11 @@
     "test": "jest --detectOpenHandles -u"
   },
   "dependencies": {
+    "@hapi/joi": "^17.1.1",
```

Comment:
Why was Unnecessary Dependency added?

Reply:
I am not sure about the reason, but don't worry I will revert that :)
Ps: I am digging deep into the issues faced by all the other students while setting up the app, and will make a guide soon for all the errors and their solutions :)